client/src/templates/Challenges/utils/build.js

```diff
@@ -145,8 +147,12 @@ export function buildDOMChallenge({
   return Promise.all(finalFiles)
     .then(checkFilesErrors)
     .then(challengeFiles => ({
-      challengeType: challengeTypes.html,
-      build: concatHtml({ required: finalRequires, template, challengeFiles }),
+      challengeType: challengeTypes.html || challengeTypes.multiFileCertProject,
```

Comment:
```suggestion
      challengeType: challengeTypes.html,
```
It always shortcuts to `challengeTypes.html`.  Does it matter if this `challengeType` is `multiFileCertProject`?

Reply:
fixed